src/main/java/de/epiceric/shopchest/ShopChest.java

```diff
@@ -99,6 +99,7 @@
     private ShopUpdater updater;
     private ExecutorService shopCreationThreadPool;
 
+
```

Comment:
unnecessary empty line

Reply:
Fix in my last commit : https://github.com/xPiston/ShopChest/commit/8ec8a63e4b8151de08dd18ba1ec1b7c140b0f0aa